src/models/StyledComponent.js

```diff
@@ -25,23 +25,13 @@ const createStyledComponent = (target: Target, rules: RuleSet) => {
     static rules: RuleSet
     static target: Target
 
-    getChildContext() {
-      return { theme: this.theme }
-    }
-
     componentWillMount() {
       this.componentWillReceiveProps(this.props, this.context)
     }
 
     componentWillReceiveProps(newProps: Object, newContext: ?any) {
       // Always pass down a theme, even if it's empty
-      this.theme = (newContext && newContext.theme) || {}
-      // Local copy for this instance with an update() method
-      const theme = Object.assign({}, this.theme, {
-        update: values => {
-          this.theme = Object.assign({}, this.theme, values)
-        },
-      })
+      const theme = (newContext && newContext.theme) || {}
```

Comment:
Huh I didn't know pure components blocked context updates. How'd you mean pass an event emitter down?